apps/services/auth-admin-api/src/app/modules/personal-representative/personal-representative.controller.ts

```diff
@@ -1,11 +1,13 @@
 import {
-  PaginatedPersonalRepresentativeRightTypeDto,
-  PersonalRepresentativeRightType,
-  PersonalRepresentativeRightTypeService,
   PersonalRepresentativeScopePermission,
   PersonalRepresentativeScopePermissionDTO,
   PersonalRepresentativeScopePermissionService,
 } from '@island.is/auth-api-lib'
+import {
+  PaginatedPersonalRepresentativeRightTypeDto,
+  PersonalRepresentativeRightType,
+  PersonalRepresentativeRightTypeService,
+} from '@island.is/auth-api-lib/personal-representative'
```

Comment:
PersonalRepresentativeScopePermission, PersonalRepresentativeScopePermissionDTO and PersonalRepresentativeScopePermissionService are not used by the personal-representative and personal-representative-public but rather by auth-admin-api so therefore we decided it should be part of the auth-api  barrel rather than personal-rep barre.